### ISSUE TITLE
Fuel Used versus FOB

### ISSUE THREAD

**Issue by kutak11** [flybywiresim/a32nx#4266]
**Mod Version**

Experimental cd80138

**Describe the bug**

Fuel used displays correct Fuel comsuption during enroute (OFP planning) but remaining FOB is not correct.
Dicrepancy between initial FOB and Used fuel.  

**To Reproduce**

1. Perform normal flight and see when on cruising level that FOB plus Fuel Used is not value of fuel when you start the flight
2.
3.

**Expected behavior**

FOB + used fuel = fuel when start the engines

**Actual behavior**

discrepancy about 15%

**References**

**Additional context**

Was this working before/when did the issue start occurring?

Is this a problem in the vanilla unmodded game?

Discord username (if different from GitHub):

**Comment by Taz5150** [comment]
Will check, but looks like the discrepancy could be due to unused fuel.

**Comment by kutak11** [comment]
> Will check, but looks like the discrepancy could be due to unused fuel.

the total "summary" of fuel FOB+Fuel used is continously increasing during flight. For example if I load 7000kg fuel on ramp, after landing Fuel Used shows 3500kg but FOB shows 4500kg. There is somehow 1000kg more FOB than it should be. All flight is prepared through Simbrief planning. Aircraft performace profile works well on Simbrief with fuel factor bias M16. Enroute Fuel used is very accurate to OFP, the only problem is that FOB is not "burning" according that.

**Comment by Taz5150** [comment]
Quick question: did you have fuel in the Center tank, or just Inner and Outer wing tanks??

**Comment by kutak11** [comment]
yes center tank empty... only Inner and outer... exactly how it loads MCDU or EFB. I do nothing with MSFS loading menu.

**Comment by Taz5150** [comment]
New fuel model has been pushed in the autopilot branch and is ready to be re-tested. Let me please know how it goes.

**Comment by MisterChocker** [comment]
Closing since no response
